meter/custom/sunspec.yaml

```diff
@@ -0,0 +1,24 @@
+type: sunspec
+params:
+- name: usage
+  choice: [ "grid", "pv", "battery" ]
+  hint: how is the meter installed
+- name: uri
+- name: id
+  default: 1
+# sample: |
+#   type: sunspec
+#   usage: {{ .usage }} # {{ .Params.Usage.Hint }}
+#   uri: {{ .uri }}
+#   uri: {{ .id }}
+render: |
+  type: modbus
+  model: sunspec
+  uri: {{ .uri }}
+  id: {{ .id }}
+  {{- if eq .usage "grid" }}
+  power: 203:W # sunspec model 203 meter
```

Comment:
Hier wird auch die Limitierung klar: für den Anwender ist es jetzt nicht mehr einfach möglich, aus 203 213 zu machen falls er nicht `float` nutzt. Lösbar wäre das über einen weiteren Parameter mit Defaultwert. Das Wissen über die Detailkonfiguration bei den Anwendern geht damit am Ende verloren...

Reply:
Wer auf der Ebene etwas anpassen will/muss, sollte sich dann in der Doku die komplexe Syntax anschauen und diese nehmen. Ich sehe hier eher den Vorteil wenn sich so ein Detail als falsch herausstellt, werden alle mit dem Update ohne Anpassungen den Fix bekommen.